CHANGELOG.md

```diff
@@ -3,11 +3,164 @@
+
+- MetroCluster dashboard should show path object metrics #746
+
+- Harvest should collect namespace resources from ONTAP #749
+
+- Harvest should be more resilient to cluster connectivity issues #480
+
+- Harvest Grafana dashboard version string should match the Harvest release #631
+
+- REST collector improvements
+  - Harvest REST collector should support ONTAP private cli endpoints #766 
+  
+  - REST collector should support ZAPI-like object prefixing #786
+
+  - REST collector should support computing new customer-defined metrics #780
+
+  - REST collector should collect aggregate, qtree and quota counters #780
+
+  - REST collector metrics should be reported in autosupport #841
+    
+  - REST collector should collect sensor counters #789
+
+  - Publish REST collector [document](https://github.com/NetApp/harvest/blob/main/docs/architecture/rest-collector.md) that highlights when you should switch from ZAPI to REST, what versions of ONTAP are supported by Harvest's REST collectors and how to fill ONTAP gaps between REST and ZAPI
+
+  - REST collector should support Qutoa, Shelf, Snapmirror, and Volume plugins #799 and #811
+
+- Improve troubleshooting and documentation on validating certificates from macOS [#723](https://github.com/NetApp/harvest/pull/723)
+
+- Harvest should read its config information from the environment variable `HARVEST_CONFIG` when supplied. This env var has higher precedence than the `--config` command-line argument. #645
+
+### Fixes
+
+- FlexGroup statistics should be aggregated across node and aggregates [#706](https://github.com/NetApp/harvest/issues/706) Thanks to @wally007 for reporting
+  
+- Network Details dashboard should use correct units and support variable sorting [#673](https://github.com/NetApp/harvest/issues/673) Thanks to @mamoep for reporting and reviewing the fix
+ 
+- Harvest Systemd service should wait for network to start [#707](https://github.com/NetApp/harvest/pull/707) Thanks to @mamoep for reporting and fixing
+
+- MetroCluster dashboard should use correct units and support variable sorting [#685](https://github.com/NetApp/harvest/issues/685) Thanks to @mamoep and @chris4789 for reporting this
+    
+- 7mode shelf plugin should handle cases where multiple channels have the same shelf id [#692](https://github.com/NetApp/harvest/issues/692) Thanks to @pilot7777 for reporting this on Slack
+
+- Improve template YAML parsing when indentation varies [#704](https://github.com/NetApp/harvest/issues/704) Thanks to @mamoep for reporting this.
+  
+- Harvest should not include version information in its container name. [#660](https://github.com/NetApp/harvest/issues/660). Thanks to @wally007 for raising this. 
+
+- Ignore missing Qtrees and improve uniqueness check on 7mode filters #782 and #797. Thanks to @jmg011 for reporting
+
+- Qtree instance key order should be the same between 7mode and cDOT #807 Thanks to @jmg011 for reporting
```

Comment:
may be we need to change the description little bit, as cmode filer response and 7mode filer response are different. And as per the Harvest instance key generation logic, instance key would be formed based on the field order in response.